components/filecoin/rs/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "filecoin-cxx"
+description = "FFI Interface to Filecoin"
+version = "0.0.1"
+edition = "2018"
+publish = false
+
+[dependencies]
+base64 = "0.13.0"
+blake2b_simd = "1.0"
+bls-signatures = { version = "0.11.3", default-features = false, features = ["pairing"] }
+cxx = { path = "../../../../third_party/rust/cxx/v1/crate"}
+fvm_shared = { version = "0.6.1 " }
+libsecp256k1 = "0.6"
+num_bigint_chainsafe = { package = "forest_bigint", version = "0.1.2"}
+serde = { version = "^1.0.117", features = ["derive"] }
+serde_json = "^1.0.59"
+thiserror = "^1.0.30"
+zx-bip44 = "0.1.0"
```

Comment:
I think we can drop this dependency. I'm not seeing where we're using it anymore